### ISSUE TITLE
[BUG] `tickfontcolor` does not allow symbolic color

### ISSUE THREAD

**Issue by ShuhuaGao** [JuliaPlots/Plots.jl#3469]
## Details
```julia
using Plots
plot(1:20, randn(20); size=(400, 300), tickfontcolor=:red)
```
The above code produces `MethodError: MethodError: no method matching alpha(::Symbol)`.

However, if we specify the color with 
```julia
plot(1:20, randn(20); size=(400, 300), tickfontcolor=colorant"red")
```
then it works as expected.

### Backends
GR
This bug occurs on ( insert `x` below )

Backend      | yes | no  | untested
-------------|-----|-----|---------
gr (default) |  x   |     |
pyplot       |     |     |x
plotly       |     |     |x
plotlyjs     |     |     |x
pgfplotsx    |     |     |x
inspectdr    |     |     |x

### Versions

Plots.jl version: v1.13.2
Backend  version (`]st -m`):  GR v0.57.4
Output of `versioninfo()`:
Julia Version 1.6.0
Commit f9720dc2eb (2021-03-24 12:55 UTC)
Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
  CPU: AMD Ryzen 9 5950X 16-Core Processor
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-11.0.1 (ORCJIT, generic)

**Comment by BeastyBlacksmith** [comment]
can't reproduce on pfgplotsx

**Comment by BeastyBlacksmith** [comment]
fixed by 096d29355046e506d118ae51798c09d46965e3a9
